applications/GeoMechanicsApplication/tests/1D-Consolidation_all_stages/README.md

```diff
@@ -0,0 +1,56 @@
+# 1D Consolidation
+
+This tests verifies that in a column of triangular plane strain elements, consolidation follows the solution given by Terzaghi in 1923.
+
+## Setup
+
+This test consists of 78 6-noded elements (using the SmallStrainUPwDiffOrderElement2D6N class) of b x h = 0.1 x 1.0 meter. A
+schematic representation can be found in the figure below:
+
+![MeshStructure](MeshStructure.svg)
```

Comment:
The figure is really good. Thanks for putting it with the documentation. Maybe just a suggestion:
Do you think it makes sense if we also show the amount of the load on top of the load? And maybe also the dimension of the soil column on the figure? I think this would help to get most of the information needed by just looking at the figure.

Reply:
tried to change according to the comments. Please check again.